tensorflow/python/ops/signal/signal.py

```diff
@@ -62,4 +62,6 @@
 from tensorflow.python.ops.signal.spectral_ops import stft
 from tensorflow.python.ops.signal.window_ops import hamming_window
 from tensorflow.python.ops.signal.window_ops import hann_window
+from tensorflow.python.ops.signal.fft_ops import fftshift
```

Comment:
In this file in the module docstring please add
`@@fftshift` and `@@ifftshift` (sorted alphabetically with the rest). This used to be how to list these functions in the tf.signal module documentation, but I'm not certain it's used anymore.